[editor]/editor_main/server/mapEditorScriptingExtension_s.lua

```diff
@@ -62,11 +62,16 @@ end
 addEventHandler("onResourceStart", resourceRoot, onResourceStartOrStop)
 addEventHandler("onResourceStop", resourceRoot, onResourceStartOrStop)
 
-function receiveLODsClientRequest()
-	triggerClientEvent(client, "setLODsClient", resourceRoot, usedLODModels)
+function onPlayerResourceStart(resourceElement)
+	local mapResource = resourceElement == resource
+
+	if not mapResource then
+		return false

```

Comment:
This function shouldn't return any _thing_.
```suggestion
		return

```